Source/SteveSloth/Private/MyEnemyManager.cpp

```diff
@@ -18,7 +18,24 @@ AMyEnemyManager::AMyEnemyManager()
 void AMyEnemyManager::BeginPlay()
 {
 	Super::BeginPlay();
-	
+
+	ULevel* currentLevel = GetWorld()->GetCurrentLevel();
+	FString levelName = currentLevel->GetOutermost()->GetName();
+
+	static ConstructorHelpers::FObjectFinder<UClass> Koala(TEXT("Blueprint'/Game/Blueprints/Enemies/EnemyProjectiles/BP_KoalaStickAmmo.BP_KoalaStickAmmo_C'"));
+	UClass* theKoala = Koala.Object;
+	FActorSpawnParameters spawnParams;
+
+	if (levelName == "LevelOne")
+	{
+		for (int i = 0; i < SpawnPoints.Num(); i++)
+		{
+			FVector SpawnLocation = SpawnPoints[i]->GetTransform().GetLocation();
+			FRotator SpawnRotation = SpawnPoints[i]->GetTransform().GetRotation().Rotator();
+
+			AActor* tryThis = GetWorld()->SpawnActor<AActor>(theKoala, SpawnLocation, SpawnRotation, spawnParams);
+		}
+	}
```

Comment:
I assume we'll have to add all the other enemies in the same way?

Reply:
I did not add this lol.. funny.. dont know who did

Reply:
this whole section?

Reply:
nevermind i know what this is.. was going to be a list then i abandoned for now until enemies are done

Reply:
i see, so should we keep this for now?